vmdb/app/presenters/menu/default_menu.rb

```diff
@@ -34,25 +34,34 @@ def clouds_menu_section
       end
 
       def infrastructure_menu_section
+        def hybrid_infra_name(name1, name2, name3)
+          lambda do
+            case EmsCluster.node_types
```

Comment:
Oh. I really just put my attention only to the menu item class based on our IRC discussion. 

If we are expected to call a method on Host class then sure, we need to pass the class too.

So I guess I shall fix that, right?